File: backend/timebank/src/main/java/kookmin/software/capstone2023/timebank/presentation/api/v1/model/bank/account/PasswordUpdateResponseData.java
package kookmin.software.capstone2023.timebank.presentation.api.v1.model.bank.account;

import java.time.LocalDateTime;

public class PasswordUpdateResponseData {

    private final String bankAccountNumber; // 계좌 번호
    private final LocalDateTime updateAt; // 변경 일자
    private final String resResultCode; // 결과 코드

    public PasswordUpdateResponseData(String bankAccountNumber, LocalDateTime updateAt, String resResultCode) {
        this.bankAccountNumber = bankAccountNumber;
        this.updateAt = updateAt;
        this.resResultCode = resResultCode;
    }

    public String getBankAccountNumber() {
        return bankAccountNumber;
    }

    public LocalDateTime getUpdateAt() {
        return updateAt;
    }

    public String getResResultCode() {
        return resResultCode;
    }
}

File: src/main/java/kookmin/software/capstone2023/timebank/presentation/api/v1/model/bank/account/PasswordUpdateResponseData.kt
package kookmin.software.capstone2023.timebank.presentation.api.v1.model.bank.account

import java.time.LocalDateTime

data class PasswordUpdateResponseData(
    val bankAccountNumber: String, // 계좌 번호
    val updateAt: LocalDateTime, // 변경 일자
    val resResultCode: String, // 결과 코드
)
